tests/src/test/java/alluxio/SystemOutRule.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+import javax.annotation.concurrent.NotThreadSafe;
```

Comment:
NIT: our import rule is like the following:
```java
import java.io.ByteArrayOutputStream;
import java.io.PrintStream;

import javax.annotation.concurrent.NotThreadSafe;
```

To get help from IDE, see http://www.alluxio.org/docs/master/en/Contributing-to-Alluxio.html#coding-style